Dshop/apps/products_catalogue/urls.py

```diff
@@ -2,13 +2,12 @@
 from django.conf.urls.static import static
 from django.urls import path
 
-from .views import CeneoProductListView, ProductListView, \
+from .views import CeneoProductListView, \
     ProductDetailView, AddToCartView, CartDetailView, CategoryListView, CategoryDetailView, DeleteOneCartItemView, \
     DeleteCartItemView
 
 
 urlpatterns = [
-    path('products_list/', ProductListView.as_view(), name='products-list'),
```

Comment:
Bring this back, but remove "products_list" so it's 
```
path('', ProductListView.as_view(), name='products-list'),
```
This way, empty url (main page) points to ProductListView and all is set in one file.